pkg/bucket/lifecycle/lifecycle.go

```diff
@@ -220,6 +220,10 @@ type ObjectOpts struct {
 	RestoreExpires   time.Time
 }
 
+func (o ObjectOpts) IsExpiredObjectDeleteMarker() bool {
```

Comment:
```suggestion
func (o ObjectOpts) ExpiredObjectDeleteMarker() bool {
```

Is enough `Is` is not needed